src/libraries/System.Diagnostics.StackTrace/src/System/Diagnostics/StackTraceSymbols.cs

```diff
@@ -201,7 +201,7 @@ void IDisposable.Dispose()
             return null;
         }
 
-        private static Stream? TryOpenFile(string? path)
+        private static Stream? TryOpenFile(string path)
```

Comment:
File.Exists doesn't accept null?

If we make this non-nullable, we can remove the `!` from the call to create a new `FileStream` on path.

Reply:
The signature was changed so that it matches the delegate required for the method call in line 192 of the same file. This is because this particular file was annotated _before_ System.Reflection.Metadata even though it depends on it.